.github/workflows/pull_requests.yml

```diff
@@ -17,4 +17,10 @@ jobs:
       with:
         version: master
 
+    # Set up OCaml
+    - name: Use OCaml 4.14.x
+    uses: ocaml/setup-ocaml@v2
```

Comment:
The indentation here is wrong

Reply:
Nice, catch will fix this in the next commit

Reply:
Fixed